internal/providers/terraform/aws/config_organization_managed_rule_test.go

```diff
@@ -0,0 +1,79 @@
+package aws_test
+
+import (
+	"testing"
+
+	"github.com/infracost/infracost/internal/providers/terraform/tftest"
+	"github.com/infracost/infracost/internal/schema"
+	"github.com/infracost/infracost/internal/testutil"
+	"github.com/shopspring/decimal"
+)
+
+func TestOrganizationManagedRuleItem(t *testing.T) {
+	if testing.Short() {
+		t.Skip("skipping test in short mode")
+	}
+
+	tf := `
+		resource "aws_config_organization_managed_rule" "my_config" {
+			name            = "example"
+			rule_identifier = "IAM_PASSWORD_POLICY"
+		}`
+
+	resourceChecks := []testutil.ResourceCheck{
+		{
+			Name: "aws_config_organization_managed_rule.my_config",
+			CostComponentChecks: []testutil.CostComponentCheck{
+				{
+					Name:      "Rule evaluations (first 100K)",
+					PriceHash: "b5643f5c83300f4a85d84a467af5aca4-3bf3a9bc78b9ee067586248fa8117ddb",
```

Comment:
```suggestion
					PriceHash: "b5643f5c83300f4a85d84a467af5aca4-3bf3a9bc78b9ee067586248fa8117ddb",
					MonthlyCostCheck: testutil.NilMonthlyCostCheck(),
```

This special `MonthlyCostCheck` ensures that the costs are nil and not 0. Please add it to the unit tests that expect this.